Cargo.toml

```diff
@@ -52,7 +52,7 @@ path = "examples/deferred/main.rs"
 name = "flowmap"
 path = "examples/flowmap/main.rs"
 
-[[target.x86_64-pc-linux-gnu.example]]
+[[example]]
```

Comment:
We still need something to test `glfw`, and `performance` does that currently. Just feature-gate it, or permanently disable on Windows.